src/styles/card.scss

```
@@ -0,0 +1,112 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 200px;
+  height: 405px;
+  padding: 32px 16px 16px;
+  margin-right: 50px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition-duration: 300ms;
```

```suggestion
  transition: hover 300ms;
```
Always add transition property. By default, transition property is all, and it's hard for the browser to control all properties.